templates/taxi/car_list.html

```diff
@@ -7,6 +8,10 @@ <h1>
       Create
     </a>
   </h1>
+  <form style="padding-bottom: 10px" action="" method="get" class="form-inline">
```

Comment:
Create a separate template for search form and include it where you need to avoid code duplication